browser/ui/toolbar/brave_location_bar_model_delegate.cc

```diff
@@ -38,6 +38,16 @@ void BraveLocationBarModelDelegate::FormattedStringFromURL(const GURL& url,
         0,
         base::UTF8ToUTF16(ethereum_remote_client_base_url),
         base::UTF8ToUTF16("brave://wallet"));
+    base::ReplaceFirstSubstringAfterOffset(
```

Comment:
I think inside this conditional we're safe to just do a full string replace to `brave://wallet`, then this is future proofed for any other routes MM may add. @bbondy thoughts on safety?

Reply:
I think this is ok as you have it. I wonder if we could just have ethereum_remote_client_base_url not have /home.html but it's not a big deal since you already have it this way.